test/show.jl

```diff
@@ -511,24 +517,139 @@ end
-          11 │   1000.0         11  false  1000.0         0.0
-          12 │  10000.0         12   true  10000.0    10000.0
-          13 │ 100000.0         13  false  100000.0  100000.0
-          14 │      1.0e6       14   true  1.0e6          1.0e6
-          15 │      1.0e7       15  false  1.0e7          1.0e7"""
+         Row │ a              b      c      d         e
+             │ Float64?       Int64  Bool   Any       Float64
+        ─────┼────────────────────────────────────────────────────
+           1 │      1.0e-7        1  false  1.0e-7         1.0e-7
+           2 │      1.0e-6        2   true  1.0e-6         1.0e-6
+           3 │      1.0e-5        3  false  1.0e-5         1.0e-5
+           4 │      0.0001        4   true  0.0001         0.0001
+           5 │      0.001         5  false  0.001          0.001
+           6 │      0.01          6   true  0.01           0.01
+           7 │      0.1           7  false  0.1            0.1
+           8 │      1.0           8   true  1.0            1.0
+           9 │     10.0           9  false  10.0          10.0
+          10 │       missing     10   true  test          -0.0
+          11 │   1000.0          11  false  1000.0         0.0
+          12 │  10000.0          12   true  10000.0    10000.0
+          13 │ 100000.0          13  false  100000.0  100000.0
+          14 │      1.0e6        14   true  1.0e6          1.0e6
+          15 │      1.0e7        15  false  1.0e7          1.0e7"""
+
+    df = DataFrame(This_is_a_very_big_name = [10.0^i for i = -5:1:5],
+                   This_is_smaller = 1.0:2:22,
+                   T = 100001:1.:100011)
+
+    io = IOBuffer()
+    show(io, df)
+    str = String(take!(io))
+    @test str == """
+        11×3 DataFrame
+         Row │ This_is_a_very_big_name  This_is_smaller  T
+             │ Float64                  Float64          Float64
+        ─────┼────────────────────────────────────────────────────
+           1 │                  1.0e-5              1.0  100001.0
+           2 │                  0.0001              3.0  100002.0
+           3 │                  0.001               5.0  100003.0
+           4 │                  0.01                7.0  100004.0
+           5 │                  0.1                 9.0  100005.0
+           6 │                  1.0                11.0  100006.0
+           7 │                 10.0                13.0  100007.0
+           8 │                100.0                15.0  100008.0
+           9 │               1000.0                17.0  100009.0
+          10 │              10000.0                19.0  100010.0
+          11 │             100000.0                21.0  100011.0"""
+
+    df = DataFrame(😄😄😄😄😄😄😄 = [10.0^i for i = -5:1:5],
+                   🚀🚀🚀🚀🚀 = 1.0:2:22)
+
+    io = IOBuffer()
+    show(io, df)
+    str = String(take!(io))
+    @test str == """
+        11×2 DataFrame
+         Row │ 😄😄😄😄😄😄😄  🚀🚀🚀🚀🚀
```

Comment:
This is a case where you're smarter than GitHub or Firefox, as it doesn't show the alignment correctly. :-D